templates/topic/index.json-ld

```diff
@@ -28,19 +29,195 @@
+			{ "title": "Context-specific features", "url": "https://wet-boew.github.io/GCWeb/components/gc-features/gc-features-doc-en.html", "component": "gc-features" },
+			{ "title": "Social media channels", "url": "https://wet-boew.github.io/GCWeb/components/gc-follow-us/gc-follow-us-doc-en.html", "component": "gc-follow-us" },
+			{ "title": "Contributors", "url": "https://wet-boew.github.io/GCWeb/components/gc-contributors/gc-contributors-doc-en.html", "component": "gc-contributors" }
+		],
+		"fr": [
+			{ "title": "Bloc d'introduction", "url": "https://wet-boew.github.io/GCWeb/design-patterns/gc-intro/gc-intro-doc-fr.html", "component": "gc-intro" },
+			{ "title": "En demande", "url": "https://wet-boew.github.io/GCWeb/components/gc-most-requested/gc-most-requested-doc-fr.html", "component": "gc-most-requested" },
+			{ "title": "Services et renseignements", "url": "https://wet-boew.github.io/GCWeb/components/gc-srvinfo/gc-srvinfo-doc-fr.html", "component": "gc-srvinfo" },
+			{ "title": "Promotions contextuelles", "url": "https://wet-boew.github.io/GCWeb/components/gc-features/gc-features-doc-fr.html", "component": "gc-features" },
+			{ "title": "Bloc des réseaux de médias sociaux", "url": "https://wet-boew.github.io/GCWeb/components/gc-follow-us/gc-follow-us-doc-fr.html", "component": "gc-follow-us" },
+			{ "title": "Collaborateurs", "url": "https://wet-boew.github.io/GCWeb/components/gc-contributors/gc-contributors-doc-fr.html", "component": "gc-contributors" }
+		]
+	},
+	"a11yGuidance": "No accessibility guidance.",
+	"variations": [
+		{
+			"name": {
+				"en": "Topic page",
+				"fr": "Page de sujet"
+			},
+			"status": "stable",
+			"description": {
+				"en": "Topic pages provide access to topics and destination pages that support task completion.",
+				"fr": "Les pages de sujet fournissent un accès aux sujets et aux pages de destination qui appuient l’achèvement des tâches."
+			},
+			"guidance": {
+				"en": "https://design.canada.ca/mandatory-templates/theme-topic.html",
+				"fr": "https://conception.canada.ca/modeles-obligatoire/theme-sujet.html"
+			},
+			"iteration": "_:iteration_topic_3",
+			"example": [
+				{
+					"en": { "href": "topic-en.html", "text": "Topic page" },
+					"fr": { "href": "topic-fr.html", "text": "Page de sujet" }
+				}
+			],
+			"implementation": [
+				"_:implement_topic",
+				"_:implement_topic_aem"
+			],
+			"history": [
+				{
+					"en": "June 2024 - Stabilization of the version 2 of the page template.",
+					"fr": "Juin 2024 - Stabilisation de la version 2 du gabarit de page."
+				},
+				{
+					"en": "January 2021 - Provisional implementation of the version 2 of the page template.",
+					"fr": "Janvier 2021 - Implémentation provisoire de la version 2 du gabarit de page."
+				},
+				{
+					"en": "August 2014 - Implementation of the version 1 of the page template.",
+					"fr": "Août 2014 - Implémentation de la version 1 du gabarit de page."
```

Comment:
The versioning of the topic page has not been well recorded until Jan 2018 and it did get a few breaking change during that period. 
May be it will be best to say: As per the template state on Jan 2018, or something similar.